src/main/java/net/openhft/chronicle/wire/WireInternal.java

```diff
@@ -48,7 +49,7 @@
     static final ThreadLocal<WeakReference<Bytes>> BYTES_TL = new ThreadLocal<>();
     static final ThreadLocal<WeakReference<Bytes>> BYTES_F2S_TL = new ThreadLocal<>();
     static final ThreadLocal<WeakReference<Wire>> BINARY_WIRE_TL = new ThreadLocal<>();
-    static final ThreadLocal<WeakReference<Bytes>> ABYTES_TL = new ThreadLocal();
+    static final ThreadLocal<WeakReference<Bytes>> INTERNAL_BYTES_TL = new ThreadLocal<>();
```

Comment:
how is this closed, resource leaks...?

Reply:
They are closed when the thread dies if it's a CleaningThread, or on a GC.

Reply:
I wonder how ThreadLocals scales with Loom looming around the corner...